VideoWeb/VideoWeb/ClientApp/src/app/shared/mic-visualiser/mic-visualiser.component.html

```diff
@@ -1 +1,3 @@
-<canvas id="meter" width="270" height="30" style="background-color: #f3f3f3;"></canvas>
+<div class="canvas-holder" #container [style.maxWidth.px]="meterMaxWidth">
```

Comment:
just double checking if this is a typo or you can actually declare ids like this in angular template files? I am sure its not I guess otherwise the test would fail

Reply:
This is for accessing elements from the ts as a ViewChild